srcpkgs/traefik/files/traefik/run

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+[ -r ./conf ] && . ./conf
+exec 2>&1
+
+exec setpriv --reuid _traefik --regid _traefik --keep-groups \
```

Comment:
Hm shouldn't this `--clear-groups` or `--init-groups` and not `--keep-groups` (keeping the `root` group)?